src/neo/SmartContract/ApplicationEngine.cs

```diff
@@ -115,6 +115,8 @@ public partial class ApplicationEngine : ExecutionEngine
         /// </summary>
         public IReadOnlyList<NotifyEventArgs> Notifications => notifications ?? (IReadOnlyList<NotifyEventArgs>)Array.Empty<NotifyEventArgs>();
 
+        private static Random random;
```

Comment:
We can't use `Random` class. Otherwise it's impossible for neo-go or other languages to implement it.

Reply:
use sha256 to replace `Random`